server/src/schema.graphql

```diff
@@ -0,0 +1,96 @@
+type Query {
```

Comment:
Great work here. It really shines how much effort and teamwork you achieved here